### DISCUSSION TITLE
Copilot inline suggestions dont work if folder name contains a newline

### DISCUSSION THREAD

**Discussion by anon-corpo-slave** [community/community#69559] [Copilot Conversations]
### Copilot not working

Copliot doesnt work if opened foldername contains a newline character.

### Description
Encountered a bug while using Copilot in VS Code. If the opened folder name contains a newline character (\n), then neither copilot inline suggestions, nor completions panel work. However copilot chat seems to work just fine. 
Removing the newline from the foldername fixes the issue, but ideally it shouldnt be an issue in the first place

OS: Ubuntu 22.04
VS Code version: 1.83.0
Copilot extension version: v1.123.471

**Comment by Akash1134** [comment]
Hi @Taha-Adeel ,

Thanks for reporting the issue. Could you please share a quick screen recording of the prompts and Copilot output? 

Also, please [share the log files](https://docs.github.com/en/copilot/troubleshooting-github-copilot/viewing-logs-for-github-copilot-in-your-environment#viewing-and-collecting-log-files) so we can understand the issue better.

Thanks!

**Comment by anon-corpo-slave** [comment]
Screenrecording of recreating the issue:
[
[Screencast from 09-10-23 06:29:31 PM IST.webm](https://github.com/community/community/assets/80640349/cac43bfb-b680-48cc-9a74-98f264230642)
](url)
